### PR TITLE
Fix appveyor

### PR DESCRIPTION
**Pull request by mloning**
#### What does this implement/fix? Explain your changes.
* Fixes stumpy>=1.5.1,<=1.9.2

#### Comments
When comparing the last successful appveyor run (https://ci.appveyor.com/project/mloning/sktime/builds/41606163/job/66bdyvexm97h7u07?fullLog=true) with the the next run after than (type hint changes) (https://ci.appveyor.com/project/mloning/sktime/builds/41607852/job/ov25h6yajbf2ibpm?fullLog=true), they both use different stumpy versions (0.9.2 and 0.10.0). 

appveyor runs used to take around 30-40 mins, they are now timing out after 1 hour.

### CODE REVIEW

**Comment by mloning** [comment]
All of the Python version are timing out.

**Comment by mloning** [comment]
With stumpy == 0.9.2, Python 3.8 succeeds. Now testing on all Python versions.

File: .appveyor.yml [hunk 1]
```diff
@@ -65,7 +66,7 @@ install:
 
   # Install requirements from inside conda environment
   - cmd: activate testenv
-  - cmd: conda install -c conda-forge "matplotlib>=3.3.2" "seaborn>=0.11.0" "tsfresh>=0.17.0" "hcrystalball>=0.1.9" "stumpy>=1.5.1" "numba==0.53" "numpy>=1.19.3" "pandas>=1.1.0,<1.2" "statsmodels==0.12.1" "scikit-learn>=0.23.0" "scikit-posthocs>=0.6.5" "pystan>=2.14,<3.0" "deprecated>=1.2.13" "fbprophet>=0.7.1"
+  - cmd: conda install -c conda-forge "matplotlib>=3.3.2" "seaborn>=0.11.0" "tsfresh>=0.17.0" "hcrystalball>=0.1.9" "stumpy==1.9.2" "numba==0.53" "numpy>=1.19.3" "pandas>=1.1.0,<1.2" "statsmodels==0.12.1" "scikit-learn>=0.23.0" "scikit-posthocs>=0.6.5" "pystan>=2.14,<3.0" "deprecated>=1.2.13" "fbprophet>=0.7.1"
```
Comment (freddyaboulton) [inline .appveyor.yml:69]: I would use stumpy>=1.5.1,<=1.9.2 to make it less restrictive. We should also do this for all operating systems to make the CI go faster!

**Review by freddyaboulton — APPROVED** [review]
@mloning Thanks for this! We should do this for all OS but approving so we can unblock AppVeyor.

**Comment by mloning** [comment]
Thanks @freddyaboulton - added the changes!

**Review by freddyaboulton — APPROVED** [review]

**Pull request merged**